builtin/credential/ldap/backend.go

```diff
@@ -227,213 +194,6 @@ func (b *backend) Login(ctx context.Context, req *logical.Request, username stri
 	return policies, ldapResponse, allGroups, nil
 }
 
-/*
- * Parses a distinguished name and returns the CN portion.
- * Given a non-conforming string (such as an already-extracted CN),
- * it will be returned as-is.
- */
-func (b *backend) getCN(dn string) string {
```

Comment:
This was moved without any internal change, to live in the client instead as it's only used there.